packages/components/spinner/src/spinner.ts

```diff
@@ -34,7 +34,7 @@ export class Spinner extends LitElement {
     super.connectedCallback();
 
     if (!this.hasAttribute('role')) {
-      this.setAttribute('role', 'alert');
+      this.setAttribute('role', 'status');
```

Comment:
Hmm, should we set a role and nothing else? Or should this be invisible to ARIA? (`role=presentation` or `aria-hidden="true"`)